### PR TITLE
Publish status when the diff isn't reviewable

### PR DESCRIPTION
**Pull request by adrianmoisey**

### CODE REVIEW

**Comment by adrianmoisey** [comment]
I did this in a rush (again), so I'm not quite sure if it's the cleanest solution. I can cleanup later if you want.

File: lintreview/review.py [hunk 1]
```diff
@@ -236,6 +236,13 @@ def publish_empty_comment(self):
         comment = IssueComment(body)
         comment.publish(self._repo, self._pr)
 
+        status = self.config.get('PULLREQUEST_STATUS', True)
```
Comment (markstory) [inline lintreview/review.py]: You could check the config in the `if` block.

**Comment by markstory** [comment]
Looks good. I would be great to get a test for this.

**Comment by adrianmoisey** [comment]
Code coverage didn't re-report after my changes

**Comment by markstory** [comment]
codecov works in mysterious ways :smile:

**Pull request merged**